Gateway/Helper/Util.php

```diff
@@ -52,7 +62,7 @@ public function __construct(
      * @param int $amount
      * @return int
      */
-    public static function formatToCents($amount = 0)
+    public static function formatToCents($amount = 0, $currencyCode = self::CURRENCY_CODE_USD)
```

Comment:
Can we update the comment to include currency code.

Reply:
is there a reason why we are passing currency code, that doesn't look like its being used

Reply:
reverted this change